cmd/createcluster.go

```diff
@@ -191,6 +197,45 @@ func runCreateCluster(w io.Writer, conf clusterConfig) error {
 		}
 	}
 
+	var depositDatas [][]byte
+	for i := 0; i < numDVs; i++ {
+		sk := secrets[i] // Group secret key for this DV
+		pubkey, err := sk.GetPublicKey()
+		if err != nil {
+			return errors.Wrap(err, "secret to pubkey")
+		}
+
+		pk, err := tblsconv.KeyToETH2(pubkey) // Group pubkey
```

Comment:
Nit: Lots of people use `pk` to refer to private key rather than public key.

Reply:
... Now I spot sk for secret key, maybe its fine as is, not sure what we use in the rest of our codebase.

Reply:
good point, i tend to restrict to using `sk` for secret keys and `pubkey` for public keys.